### ISSUE TITLE
v2: Add version selector to migrate to v1 docs

### ISSUE THREAD

**Issue by endigo9740** [skeletonlabs/skeleton#1895]
### Link to the Page

(fixed in header)

### Describe the Issue (screenshots encouraged!)

Now that v2 is about to drop, we need a simple way to navigate back to the v1 documentation. To handle this I'm going to implement a version selector in the App Bar at the top of the doc site. This will be modeled after options like the Vite website:

![Screenshot 2023-08-17 at 3 03 37 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/8d50bd4a-76d1-4216-9827-f503ce1ac32a)

Note that this will not be updated retroactively for prior version of the doc site. But this should be fine. Essentially:

v4 => includes links to v3, v2, v1
v3 => includes links to v2, v1
v2 => includes links to v1
v1 => no prior version to link to

So to summarize, prior versions will not redirect to new versions unless we keep the "New version available" banner on the homepage for prior versions.

**Comment by endigo9740** [comment]
Here's a preview for the Skeleton iteration with minor update to the App Bar navigation layout:

![Screenshot 2023-08-17 at 3 47 26 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/8cdcba4c-341a-41a7-9c92-1c3c88872d72)
